spec.emu

```diff
@@ -268,93 +272,231 @@ contributors: Ujjwal Sharma, Younies Mahmoud
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-partitiondurationformatpattern" type="abstract operation">
+    <emu-clause id="sec-setfractionaldigitoptions" type="abstract operation">
       <h1>
-        PartitionDurationFormatPattern (
-          _durationFormat_: a DurationFormat,
+        SetFractionalDigitOptions (
+        _durationFormat_: a DurationFormat Object,
```

Comment:
The only thing you use internally is _durationFormat_.[[FractionalDigits]], why not just pass in that value and avoid passing in _durationFormat_ ?